Sources/CodegenCLI/Commands/Initialize.swift

```diff
@@ -86,11 +82,11 @@ public struct Initialize: ParsableCommand {
 
   func _run(fileManager: ApolloFileManager = .default, output: OutputClosure? = nil) throws {
     let encoded = try ApolloCodegenConfiguration
-      .minimalJSON(
-        schemaName: schemaName,
-        moduleType: moduleType,
-        targetName: targetName
-      ).asData()
+      .minimalJSON(schemaName: schemaName, moduleType: moduleType, targetName: targetName)
+      .asData()
+
+    let decoded = try JSONDecoder().decode(ApolloCodegenConfiguration.self, from: encoded)
```

Comment:
I'm not thrilled about decoding the config data but it's a small price to pay considering this is an infrequent workflow, not in the critical path of performance and IMO it's far better than having one single public method that only validates the schema name string.